cogboard-webapp/src/components/BoardForm/index.js

```diff
@@ -11,6 +11,18 @@ const BoardForm = ({ renderActions, ...initialFormValues }) => {
   const { values, handleChange } = useFormData(initialFormValues);
   const { autoSwitch } = values;
 
+  const handleInput = (e) => {
+    const inputValue = e.target.value;
+    let intValue = isNaN(parseInt(inputValue)) ? COLUMNS_MIN : parseInt(inputValue);
+    intValue = intValue > COLUMNS_MAX ? COLUMNS_MAX : intValue;
+    intValue = intValue < COLUMNS_MIN ? COLUMNS_MIN : intValue;
+    e.target.value = intValue.toString();
+  };
+
+  const columnsPlaceholder = () => {
```

Comment:
I think we don't need a function here, but if you want it to be a function, please move it from component to helpers, and change name to getColumnsPlaceholder.